soap/src/java/com/zimbra/soap/mail/type/Folder.java

```diff
@@ -279,6 +279,13 @@ public static View fromString(String name) {
     private String restUrl;
 
     /**
+     * @zm-api-field-tag deletable
+     * @zm-api-field-description whether this folder can be deleted
+     */
+    @XmlAttribute(name=MailConstants.A_DELETABLE /* deletable */, required=false)
+    private boolean deletable;
```

Comment:
Check this commit if you want a cheatsheet: 6f28fd1a4b2f2070383a7d5171b269b7eb82428b